.gitignore

```diff
@@ -37,3 +37,4 @@ log
 test/tmp/*
 *.gem
 Gemfile.lock
+*.lock
```

Comment:
What's this for?

Reply:
Appraisal generates Gemfiles in the /gemfiles directories.  We don't want to check in their lock files.